client/components/Parts/Trees/File.tsx

```diff
@@ -93,21 +102,96 @@ const StyledTreeItem = styled(
       ['create', 'delete', 'update'].includes(event.type)
         ? `${fileEventKeyframe} 1s`
         : undefined
+
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+    const open = Boolean(anchorEl);
+    const handleContextBtnClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+      setAnchorEl(event.currentTarget);
+    }
+    const handleClose = () => {
+      setAnchorEl(null);
+    }
+
+    const handleDelete = () => {
+      store.openDialog('deleteFilesFolders')
+      handleClose()
+    }
+
+    const handleOpenFileLocation = () => {
+      // window.opendataeditor.openPathInExplorer(path)
```

Comment:
@roll I need help getting the full path of the file. Do you know how to do it? :pray: